src/main/java/com/hyperwallet/clientsdk/model/DocumentVerificationReason.java

```diff
@@ -0,0 +1,23 @@
+package com.hyperwallet.clientsdk.model;
+
+public enum DocumentVerificationReason {
+    DOCUMENT_EXPIRED("Document in expired"),
```

Comment:
we have a typo in our HW2 code also

Reply:
should be something like "Document is/has expired"

Reply:
Have asked Ash about it,  if it needs to be fixed in hw2 code, let's see what he says.

Reply:
Will create a seperate ticket for this after confirming with Ash, once he is back.